Jenkinsfile

```diff
@@ -53,7 +53,7 @@ pipeline {
                         dir("${env.DOCKER_BUILD_DIR}/test/intergov/") {
                             sh '''#!/bin/bash
                                 export COMPOSE_PROJECT_NAME=au
-                                cp demo-au.env demo-au.env
+                                cp demo-au.env demo-au-local.env
```

Comment:
please note these files are different. demo-au-local is fine to be empty by default (mine contains IGL_OAUTH_CLIENT_ID and other things to work with the shared DB channel deployed by Michael)

so it's fine to create empty demo-*-local.env file; but copying works fine for the same reason - just re-defines variables in their same values

Reply:
Don't we need to at least define `IGL_COUNTRY=AU`? If not now, I suspect that it will be mandatory for a node to have that explicitly defined.

Anyway, as we make clearer the distinction between just spinning up one or more nodes and running a specific set of nodes and channels for the purpose of running integration tests, I expect this code to change.

Reply:
![Screenshot from 2020-05-21 09-11-51](https://user-images.githubusercontent.com/131850/82533229-2723e780-9b43-11ea-91e8-d4e2eb6ad195.png)

look, all 3 files are used. so demo-au.env contains exactly that, and demo-au-local contains whatever you need for you locally.

for integration tests insinde the single country we just start up any of it (au, cn). integration tests taking 2 or more countries will require starting 2.

Reply:
Ahhhh, sorry, yes, I see!